docs/data/routes/docs/nextjs/data-fetching/component-level-data-fetching/en.md

```diff
@@ -3,4 +3,87 @@ name: component-level-data-fetching
 routeTemplate: ./data/component-templates/article.yml
 title: Component-level data fetching
 ---
-# Component-level data fetching
\ No newline at end of file
+# Component-level data fetching
+
+Next.js provides ability to pre-render pages at build time using [getStaticProps](https://nextjs.org/docs/basic-features/data-fetching#getstaticprops-static-generation). Expected that you will fetch all required data in `getStaticProps`.
```

Comment:
Should we mention `getServerSideProps` here too? It has the same limitation, you must fetch all data for a route up front.

Reply:
@nickwesselman Good call, I mentioned it